lib/cc/analyzer/engines_runner.rb

```diff
@@ -85,7 +85,15 @@ def registry_lookup(engine_name)
       end
 
       def exclude_paths
-        PathPatterns.new(@config.exclude_paths || []).expanded
+        PathPatterns.new(@config.exclude_paths || []).expanded + gitignore_paths
+      end
+
+      def gitignore_paths
+        if File.exist?(".gitignore")
+          `git ls-files --others -i --exclude-from .gitignore`.split("\n")
```

Comment:
Do you know if `ls-files` has an argument to delimit these with `\0`? Other git calls have that